2016-12-06 18:02:58 Error: Fatal Error (64): Cannot redeclare Note::$belongsTo in [C:\wamp\www\moo2\app\Plugin\Note\Model\Note.php, line 20]
2016-12-06 18:02:58 Error: Fatal Error (64): Cannot redeclare Note::$belongsTo in [C:\wamp\www\moo2\app\Plugin\Note\Model\Note.php, line 20]
2016-12-06 18:02:58 Error: [FatalErrorException] Cannot redeclare Note::$belongsTo
Request URL: /moo2/notes
Stack Trace:
#0 C:\wamp\www\moo2\lib\Cake\Error\ErrorHandler.php(213): ErrorHandler::handleFatalError(64, 'Cannot redeclar...', 'C:\\wamp\\www\\moo...', 20)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot redeclar...', 'C:\\wamp\\www\\moo...', 20, Array)
#2 C:\wamp\www\moo2\lib\Cake\Core\App.php(933): call_user_func('ErrorHandler::h...', 64, 'Cannot redeclar...', 'C:\\wamp\\www\\moo...', 20, Array)
#3 C:\wamp\www\moo2\lib\Cake\Core\App.php(906): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-12-06 18:02:58 Error: [FatalErrorException] Cannot redeclare Note::$belongsTo
Request URL: /moo2/notes
Stack Trace:
#0 C:\wamp\www\moo2\lib\Cake\Error\ErrorHandler.php(213): ErrorHandler::handleFatalError(64, 'Cannot redeclar...', 'C:\\wamp\\www\\moo...', 20)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot redeclar...', 'C:\\wamp\\www\\moo...', 20, Array)
#2 C:\wamp\www\moo2\lib\Cake\Core\App.php(933): call_user_func('ErrorHandler::h...', 64, 'Cannot redeclar...', 'C:\\wamp\\www\\moo...', 20, Array)
#3 C:\wamp\www\moo2\lib\Cake\Core\App.php(906): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-12-06 18:09:15 Error: Fatal Error (1): Class 'NoteTaskReminder' not found in [C:\wamp\www\moo2\app\Plugin\Cron\Model\Task.php, line 456]
2016-12-06 18:09:15 Error: Fatal Error (1): Class 'NoteTaskReminder' not found in [C:\wamp\www\moo2\app\Plugin\Cron\Model\Task.php, line 456]
2016-12-06 18:09:15 Error: [FatalErrorException] Class 'NoteTaskReminder' not found
Request URL: /moo2/cron/task/run?key=3FSE%40
Stack Trace:
#0 C:\wamp\www\moo2\lib\Cake\Error\ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'NoteTask...', 'C:\\wamp\\www\\moo...', 456)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'NoteTask...', 'C:\\wamp\\www\\moo...', 456, Array)
#2 C:\wamp\www\moo2\lib\Cake\Core\App.php(933): call_user_func('ErrorHandler::h...', 1, 'Class 'NoteTask...', 'C:\\wamp\\www\\moo...', 456, Array)
#3 C:\wamp\www\moo2\lib\Cake\Core\App.php(906): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-12-06 18:09:15 Error: [FatalErrorException] Class 'NoteTaskReminder' not found
Request URL: /moo2/cron/task/run?key=3FSE%40
Stack Trace:
#0 C:\wamp\www\moo2\lib\Cake\Error\ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'NoteTask...', 'C:\\wamp\\www\\moo...', 456)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'NoteTask...', 'C:\\wamp\\www\\moo...', 456, Array)
#2 C:\wamp\www\moo2\lib\Cake\Core\App.php(933): call_user_func('ErrorHandler::h...', 1, 'Class 'NoteTask...', 'C:\\wamp\\www\\moo...', 456, Array)
#3 C:\wamp\www\moo2\lib\Cake\Core\App.php(906): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-12-06 18:25:23 Error: Fatal Error (1): Class 'NoteTaskReminder' not found in [C:\wamp\www\moo2\app\Plugin\Cron\Model\Task.php, line 456]
2016-12-06 18:25:23 Error: Fatal Error (1): Class 'NoteTaskReminder' not found in [C:\wamp\www\moo2\app\Plugin\Cron\Model\Task.php, line 456]
2016-12-06 18:25:23 Error: [FatalErrorException] Class 'NoteTaskReminder' not found
Request URL: /moo2/cron/task/run?key=3FSE%40
Stack Trace:
#0 C:\wamp\www\moo2\lib\Cake\Error\ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'NoteTask...', 'C:\\wamp\\www\\moo...', 456)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'NoteTask...', 'C:\\wamp\\www\\moo...', 456, Array)
#2 C:\wamp\www\moo2\lib\Cake\Core\App.php(933): call_user_func('ErrorHandler::h...', 1, 'Class 'NoteTask...', 'C:\\wamp\\www\\moo...', 456, Array)
#3 C:\wamp\www\moo2\lib\Cake\Core\App.php(906): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-12-06 18:25:23 Error: [FatalErrorException] Class 'NoteTaskReminder' not found
Request URL: /moo2/cron/task/run?key=3FSE%40
Stack Trace:
#0 C:\wamp\www\moo2\lib\Cake\Error\ErrorHandler.php(213): ErrorHandler::handleFatalError(1, 'Class 'NoteTask...', 'C:\\wamp\\www\\moo...', 456)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'NoteTask...', 'C:\\wamp\\www\\moo...', 456, Array)
#2 C:\wamp\www\moo2\lib\Cake\Core\App.php(933): call_user_func('ErrorHandler::h...', 1, 'Class 'NoteTask...', 'C:\\wamp\\www\\moo...', 456, Array)
#3 C:\wamp\www\moo2\lib\Cake\Core\App.php(906): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
